src/main/features/response/routes/response-type.ts

```diff
@@ -36,9 +36,9 @@ export default express.Router()
         await ResponseDraftMiddleware.save(res, next)
 
         if (ResponseType.OWE_NONE === form.model.type) {
-          res.redirect(Paths.defenceOptionsPage.evaluateUri({ externalId: externalId }))
```

Comment:
this check isn't right, there's 3 pages it could go to:
part admission
full rejection
task-list

I just pulled your branch and the full admission button is going to partial admission